ledger/src/shred.rs

```diff
@@ -682,21 +680,29 @@ impl TryFrom<u8> for ShredVariant {
 pub fn should_discard_shred(
     packet: &Packet,
     root: Slot,
+    max_slot: Slot, // Max slot to ingest shreds for.
```

Comment:
Given that the lower/upper bound check are now separate, what do you think about moving this comment to above function declaration, and also mention `root`, maybe something like:
```
// Accept shreds in the slot range [root + 1, max_slot]
```

Reply:
done.